docs/src/main/sphinx/connector/hive-alluxio.rst

```diff
@@ -1,91 +1,21 @@
-a resource. Update the Trino :ref:`jvm-config` file ``etc/jvm.config``
-to include the following:
-
-.. code-block:: text
-
-  -Xbootclasspath/a:<path-to-alluxio-conf>
-
-The advantage of this approach is that all the Alluxio properties are set in
-the single ``alluxio-site.properties`` file. For details, see `Customize Alluxio Presto Properties
-<https://docs.alluxio.io/os/user/2.1/en/compute/Presto.html#customize-alluxio-user-properties?utm_source=trino&utm_medium=trinodocs>`_.
-
-Alternatively, add Alluxio configuration properties to the Hadoop configuration
-files (``core-site.xml``, ``hdfs-site.xml``) and configure the Hive connector
-to use the `Hadoop configuration files <#hdfs-configuration>`__ via the
-``hive.config.resources`` connector property.
-
-Deploy Alluxio with Trino
---------------------------
-
-To achieve the best performance running Trino on Alluxio, it is recommended
-to collocate Trino workers with Alluxio workers. This allows reads and writes
-to bypass the network (*short-circuit*). See `Performance Tuning Tips for Presto with Alluxio
-<https://www.alluxio.io/blog/top-5-performance-tuning-tips-for-running-presto-on-alluxio-1/?utm_source=trino&utm_medium=trinodocs>`_
-for more details.
-
-.. _alluxio-catalog-service:
-
-Alluxio catalog service
------------------------
-
-An alternative way for Trino to interact with Alluxio is via the
-`Alluxio catalog service <https://docs.alluxio.io/os/user/stable/en/core-services/Catalog.html?utm_source=trino&utm_medium=trinodocs>`_.
-The primary benefits for using the Alluxio catalog service are simpler
-deployment of Alluxio with Trino, and enabling schema-aware optimizations
-such as transparent caching and transformations. Currently, the catalog service
-supports read-only workloads.
-
-The Alluxio catalog service is a metastore that can cache the information
-from different underlying metastores. It currently supports the Hive metastore
-as an underlying metastore. In order for the Alluxio catalog to manage the metadata
-of other existing metastores, the other metastores must be "attached" to the
-Alluxio catalog. To attach an existing Hive metastore to the Alluxio
-catalog, simply use the
-`Alluxio CLI attachdb command <https://docs.alluxio.io/os/user/stable/en/operation/User-CLI.html?utm_source=trino&utm_medium=trinodocs#attachdb>`_.
-The appropriate Hive metastore location and Hive database name need to be
-provided.
-
-.. code-block:: text
-
-    ./bin/alluxio table attachdb hive thrift://HOSTNAME:9083 hive_db_name
-
-Once a metastore is attached, the Alluxio catalog can manage and serve the
-information to Trino. To configure the Hive connector for Alluxio
-catalog service, simply configure the connector to use the Alluxio
-metastore type, and provide the location to the Alluxio cluster.
-For example, your ``etc/catalog/alluxio.properties`` should include
-the following:
-
-.. code-block:: text
-
-    connector.name=hive
-    hive.metastore=alluxio-deprecated
```

Comment:
The redirected page doesn't contain this property key and value. I don't think users can setup the environment with the Alluxio page.

Reply:
From what we understand talking to the Alluxio team this is no longer needed. @beinan ?

Reply:
Yes, we will remove the code of "alluxio-deprecated" metastore

Reply:
Sure .. but is it needed now still? With the current integration documented on the Alluxio docs?

Reply:
> we will remove the code of "alluxio-deprecated" metastore

Do you mean the connector is going to remove support for Alluxio metastore? It would be nice to add `warning::` message here if so.

Reply:
From what I understand this is no longer necessary .. but @beinan is checking to make sure.

Reply:
#18195